easybuild/easyconfigs/p/polars/polars-1.27.1_polars-stream.patch

```diff
@@ -0,0 +1,210 @@
+Fix polars-stream crate.
```

Comment:
Where does fix come from (and what does it fix exactly, any relevant error messages we can mention)?

Any reference we can add here (PR, issue, etc.)?